.github/workflows/visual-tests.yml

```
@@ -0,0 +1,83 @@
+name: Visual Tests
+
+on: 
+  workflow_dispatch:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
+  
+
+ # Allow updating snapshots during manual runs
+  workflow_call:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
+
+jobs:
+  visual-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        project:
+          - name: Platform
+          - name: Install
+    env:
+      SHOPWARE_HTTP_CACHE_ENABLED: 0
+      SHOPWARE_DISABLE_UPDATE_CHECK: "true"
+      BLUE_GREEN_DEPLOYMENT: 1
+      COMPOSER_ROOT_VERSION: 6.6.9999999-dev
+    steps:
+      - name: Setup Shopware
+        uses: shopware/setup-shopware@main
+        with:
+          shopware-version: ${{ github.ref }}
+          shopware-repository: ${{ github.repository }}
+          install: ${{ matrix.project.name != 'Install' }} # When testing the installation routine of Shopware, don't execute the installation automatically
+          install-admin: ${{ matrix.project.name != 'Install' }}
+          install-storefront: ${{ matrix.project.name != 'Install' }}
+          env: prod
+
+      - name: Build js
+        run: |
+          mkdir -p config/jwt
+          composer run build:js:admin
+          composer npm:storefront run production # we cannot use build:js because it requires a db
+          (cd src/Storefront/Resources/app/storefront && node copy-to-vendor.js)
+      - name: Start web server
+        run: symfony server:start -d
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        working-directory: tests/acceptance
+        run: npm ci
+      - name: Install Playwright Browsers
+        working-directory: tests/acceptance
+        run: npx playwright install --with-deps chromium
+      - name: Set up cache
+        id: cache
+        uses: actions/cache@v4
+        with:
+          key: cache/${{github.repository}}/${{github.ref}}
+          restore-keys: cache/${{github.repository}}/refs/heads/trunk
+          path: .test/**
+      - name: Initialize snapshots
+        if: ${{steps.cache.outputs.cache-hit != 'true' || inputs.update-snapshots == 'true'}}
+        run: npx playwright test --grep "@visual-test" --update-snapshots --reporter html
+      - name: Run your tests
+        continue-on-error: true
+        env:
+            CURRENTS_PROJECT_ID: ${{ secrets.CURRENTS_PROJECT_ID }}
+            CURRENTS_RECORD_KEY: ${{ secrets.CURRENTS_RECORD_KEY }}
+        if: ${{ !contains(github.workflow_ref, 'nightly') }}
+        working-directory: tests/acceptance
+        run: npx playwright test  --grep "@visual-test" --project=${{ matrix.project.name }} ${{ matrix.project.name == 'Install' && '--trace=on' || '' }}
```

```suggestion
        run: npx playwright test --project=Visual
```

Please also edit the playwright config